frontend/apps/reader/modules/readergoto.lua

```diff
@@ -63,6 +90,20 @@ function ReaderGoto:onShowGotoDialog()
     UIManager:show(self.goto_dialog)
 end
 
+function ReaderGoto:onShowSkimtoDialog()
+    self.skimto = SkimToWidget:new{
+        document = self.document,
+        callback_goto_page = function(page)
```

Comment:
Why create this callback when we can call `self.ui:handleEvent` directly from skim widget?